addon/templates/components/materialize-button.hbs

```diff
@@ -2,3 +2,4 @@
   <i {{bind-attr class="icon iconPosition"}}></i>
 {{/if}}
 {{text}}
+{{yield}}
```

Comment:
This is a tiny bugfix to get block syntax working with button
```js
{{#materialize-button class="btn btn-flat"}}
My Label
{{/materialize-button}}
```
Which is currently broken